### ISSUE TITLE
Regarding the Parameter Quantity and Accuracy of the EAT-S Model

### ISSUE THREAD

**Issue by fyl-yyds** [Alibaba-MIIL/AudioClassfication#18]
Hello
I think your paper is great, but I encountered some problems during training. I hope you can help me take a look.
When I used ESC-50 to train EAT-S on a single 4090，The default parameters used are README.md:
Fs=22.05KHz\
seq_len = 114688 ~5sec\
python trainer.py --max_lr 3e-4 --run_name r1 --emb_dim 128  --dataset esc50 --seq_len 114688  --mix_ratio 1 --epoch_mix 12 --mix_loss bce --batch_size 128 --n_epochs 3500 --ds_factors 4 4 4 4 --amp --save_path outputs\
My model has a parameter size of around 5.18M and a 5-fold cross validation accuracy of 90.5, which is not consistent with the 5.3M and 92.15 values mentioned in the paper. Do you know what the problem is ?

**Comment by avi33** [comment]
Hey, 
sorry for the late response - yes, it is not consistent to the reported result when trained from scratch. Unforrtunely our site was closed so i cannot reproduce the training result so i will try on my own computer, so it will take some time..
is it the official split? (i supose it is just verifying)
did you used EMA during training? the default shall be enabled
